src/screens/OrganizationFunds/OrganizationFunds.tsx

```
@@ -246,7 +208,7 @@ const organizationFunds = (): JSX.Element => {
       sortable: false,
       headerClassName: `${styles.tableHeader}`,
       renderCell: (params: GridCellParams) => {
-        return params.row.creator.name;
+        return params.row.creator.firstName + ' ' + params.row.creator.lastName;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)